xml/depl_crowbar.xml

```diff
@@ -0,0 +1,327 @@
+    <varlistentry>
+     <term><guimenu>&ostack;</guimenu></term>
+     <listitem>
+      <para>This screen only shows the &barcl;s that are dedicated to &ostack;
+       service deployment and configuration.  From here, you can
+       <guimenu>Edit</guimenu> individual &barcl;
+       proposals. </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><guimenu>Deployment Queue</guimenu></term>
+     <listitem>
+      <para>If &barcl;s are applied to one or more nodes that are nor yet
+       available for deployment (for example, because they are rebooting or have
+       not been fully installed yet), the proposals will be put in a queue. This
+       screen shows the proposals that are <guimenu>Currently deploying</guimenu>
+       or <guimenu>Waiting in queue</guimenu>.</para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect2>
+  
+  <sect2 xml:id="sec.depl.crow.overview.utilities">
+   <title>Utilities</title>
+   <variablelist>
+    <varlistentry>
+     <term><guimenu>Exported Items</guimenu></term>
+     <listitem>
+      <para>The <guimenu>Exported Files</guimenu> screen allows you to export the
+       &chef; configuration and the <literal>supportconfig</literal> TAR
+       archive. The <literal>supportconfig</literal> archive contains system
+        information such as the current kernel version being used, the hardware, RPM
+        database, partitions, and the most important log files for analysis of any
+        problems. To access the export options, click <guimenu>New
+        Export</guimenu>. After the export has been successfully finished, the
+        <guimenu>Exported Files</guimenu> screen will show any files that are
+       available for download.</para>
+     </listitem>
+    </varlistentry>
+    <!--taroth 2016-02-05: https://bugzilla.suse.com/show_bug.cgi?id=952643-->
+    <varlistentry>
+     <term><guimenu>Repositories</guimenu></term>
+     <listitem>
+      <para>This screen shows an overview of the mandatory, recommended and
+       optional repositories for all architectures of &productname;. On each
+       reload of the screen the &crow; &wi; checks the availability and
+       status of the repositories. If a mandatory repository is not present, it
+       is marked red in the screen. Any repositories marked green are usable and
+       available to each node in the cloud. If you activate the check box for a
+       repository in the <guimenu>Active</guimenu> column, the managed nodes
+       will automatically be configured to use this repository. <remark>taroth
+        2016-02-05: DEVs: and if the check box is *not* ticked? will the nodes
+        use another repo? or no repo at all? and what's the default - no repo
+        activated?</remark></para>
+      <para>You cannot edit any repositories in this screen. The list of
+       repositories shown here is defined in
+        <filename>/etc/crowbar/repos.yml</filename>. If you need additional,
```

Comment:
@MaximilianMeister: Thanks! I rephrased the sentence about /etc/crowbar/repos.yml. However, I would not want to use the example from mkcloud since it points to an internal URL.  
@vuntz, @MaximilianMeister: In the test deployment I'm looking at; i found the following:
```suse-12.1:
  x86_64:
    sles12-sp1-pool:
      name: "SLES12-SP1-Pool"
      required: "mandatory"
      features: ["os"]
      repomd:
        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:GA/SLES/12.1/POOL/x86_64"
        key_md5: "9a62177e1c6852d48453ed3909956d6b"
      url:
      smt_path: "SUSE/Products/SLE-SERVER/12-SP1/x86_64/product"
      ask_on_error: false
```
Please let me know if we can use that one as an example (it's taken from /opt/dell/crowbar_framework/config/repos-cloud.yml, however) or what I need to remove for a minimal example (only containing the minimum required settings).